metrics_utility/automation_controller_billing/collectors.py

```diff
@@ -551,10 +552,17 @@ def total_workers_vcpu(since, full_path, until, **kwargs):
         raise MissingRequiredEnvVar('environment variable METRICS_UTILITY_CLUSTER_NAME is not set')
 
     now = datetime.now(timezone.utc)
-
-    info = {'cluster_name': cluster_name, 'timestamp': now.isoformat(), 'nodes': []}
-    # If METRICS_UTILITY_USAGE_BASED_BILLING_ENABLED is not set or set to false then it returns 1
-    usage_based_billing_enabled_str = os.getenv('METRICS_UTILITY_USAGE_BASED_BILLING_ENABLED')
+    current_ts = now.timestamp()
+    prev_hour_start, prev_hour_end = get_hour_boundaries(current_ts)
+
+    info = {
+        'cluster_name': cluster_name,
+        'collection_timestamp': datetime.fromtimestamp(current_ts).isoformat(),
+        'start_timestamp': datetime.fromtimestamp(prev_hour_start).isoformat(),
+        'end_timestamp': datetime.fromtimestamp(prev_hour_end).isoformat(),
```

Comment:
```suggestion
        'collection_timestamp': datetime.fromtimestamp(current_ts, timezone.utc).isoformat(),
        'start_timestamp': datetime.fromtimestamp(prev_hour_start, timezone.utc).isoformat(),
        'end_timestamp': datetime.fromtimestamp(prev_hour_end, timezone.utc).isoformat(),
```

Unless these are intentionally timezone-less in server-local timezone?

Reply:
this is a good concern and so I doubled check our output to splumk and what we send to analytics... all are UTC and it is what we want.
@robinbobbitt confirmed it.
Thx
I think we are good for an approval

Reply:
> I think we are good for an approval

You got it :) Approved